Extensions/ParticleSystem/particleemitterobject.ts

```diff
@@ -403,6 +403,10 @@ namespace gdjs {
       return (this.angleA + this.angleB) / 2.0;
     }
 
+    getRendererType(): float {
+      return parseInt(this.rendererType, 10) || 0;
```

Comment:
Did you check what's inside rendererType? It's a string which is "Line", "Point" or "Quad" so I'm unsure this was working at all?